harvard-huit/Harvard-Geospatial-Library
# Adding nginx configuration file.

**Adding nginx configuration file.**
* * *

# What does this Pull Request do?
This adds a default nginx configuration file that can be mapped on each server. This way we can add additional nginx settings such as `client_body_timeout` and `client_header_timeout`

# How should this be tested?

A description of what steps someone could take to:
* After pulling in the branch, you should see a new `hgl.conf.template` in your project root
* Copy `hgl.conf.template` to `hgl.conf`
* You do not need to change any settings. Just rebuild your container.
* HGL should come up and run normally, but with those additional nginx configurations in place.
* You can bash into the container as root (`docker exec -it --user root geoblacklight_web bash`) and then confirm that the `/etc/nginx/conf.d/hgl.conf` file exists in the container and has the same settings as your local file if you wish.
* 
# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No
